src/material-ui-theme.tsx

```diff
@@ -26,7 +26,7 @@ export default createMuiTheme({
         },
         "&$disabled": {
           color: "inherit",
-          opacity: 0.25
+          opacity: 0.5
```

Comment:
This change affected start/stop button, that should be 0.25 while disabled. I know that it's 0.5, so the season button gets 0.5. Also, note "Season" label should be 0.25, not 0.5 too. 
A bit of nitpicking obviously, but since Micheal put some work to come up with all these values, I think it'd be nice to follow them if we can (and this shouldn't be hard to tweak). 😉